### PR TITLE
Add more info to location of external sources

### PR DESCRIPTION
**Pull request by pedrorijo91**
Add more info to location of external sources on issue template

As mentioned in #962, it is not clear to new users that some website content is extracted from other sources like the progit book and git man pages. This tries to improve a little but I think it could still be improved, but I don't have any good idea for now

### CODE REVIEW

**Comment by pedrorijo91** [comment]
cc @peff and @sxlijin

**Comment by sxlijin** [comment]
Maybe something like this?


- [ ] This issue is about **specifically** the [git-scm.com](https://git-scm.com) website and is **not** an issue about
    - the Git documentation (a.k.a. man/help pages, i.e. anything with a URL starting with `https://git-scm.com/docs`), which should be raised with the [community](https://git-scm.com/community),
    - the contents of the Pro Git book (i.e. anything with a URL starting with `https://git-scm.com/book` or its PDF versions), which should be raised at [progit/progit2](https://github.com/progit/progit2/issues),
    - Git itself, which should also be raised with the [community](https://git-scm.com/community), or
    - Git for Windows, which should be raised at [git-for-windows/git](https://github.com/git-for-windows/git).

@StingyJack do you think something like this would have been clearer when you opened #962?

**Comment by pedrorijo91** [comment]
Just waiting to see if @StingyJack has any thoughts on this before updating the PR

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
- - [ ] This issue is about **specifically** the [git-scm.com](https://git-scm.com) website and is **not** an issue about
-     - [ ] Git (or its man pages), which should be raised with the [community](https://git-scm.com/community),
+- [ ] This issue is about **specifically** the [git-scm.com](https://git-scm.com) website and is **not** an issue about
+     - [ ] Git (or its man pages), which are located under https://git-scm.com/docs, should be raised with the [community](https://git-scm.com/community),
```
Comment (peff) [inline .github/ISSUE_TEMPLATE.md]: Should this new phrase go inside the parenthetical with "man pages"? I.e.:

    Git (or its man pages, which are located under https://git-scm.com/docs), which should be raised...

Maybe "mirrored" would be better than "located", too, to make it clear that we are pulling in the content.

**Comment by peff** [comment]
Oh, I should have read all the comments before the diff. The text from @sxlijin looks OK to me, too.

**Comment by StingyJack** [comment]
The addition of the link is good. But the first line is the problem... 

>"this is an issue with the website.."

... yep, it is, and I've probably stopped reading the rest.

**Comment by pedrorijo91** [comment]
maybe adding an introductory note as:

```
The git website combines information from several sources. Make sure the problem you are reporting is from the website itself and not from one of the external resources or git itself:

- git man pages (....)
- git book (...)
- git (unexpected behaviour or feature request), which should be raised with the community (...)
- git for windows, which has a parallel development (...)

If you still think the problem you are facing is related with this repository, please describe your problem as detailed as possible (including links and etc)
```

**Comment by sxlijin** [comment]
New suggestion below:

- [ ] This is **not** an issue about
    - the Git documentation (a.k.a. man/help pages, i.e. anything with a URL starting with `https://git-scm.com/docs`), which should be raised with the [community](https://git-scm.com/community),
    - the contents of the Pro Git book (i.e. anything with a URL starting with `https://git-scm.com/book` or its PDF versions), which should be raised at [progit/progit2](https://github.com/progit/progit2/issues),
    - Git itself, which should also be raised with the [community](https://git-scm.com/community), or
    - Git for Windows, which should be raised at [git-for-windows/git](https://github.com/git-for-windows/git),
    - and is actually about the website itself (e.g. JS or CSS that doesn't seem to be doing its job correctly).

@StingyJack?

**Comment by peff** [comment]
@sxlijin I like that wording, though I'd probably drop the final one (or make sure to mention "content" in there alongside JS and CSS, but this really is the right place to talk about anything found on the website that isn't from the imported sources).

**Comment by sxlijin** [comment]
How about:

- [ ] This is **not** an issue about
    - the Git documentation (a.k.a. man/help pages, i.e. anything with a URL starting with `https://git-scm.com/docs`), which should be raised with the [community](https://git-scm.com/community),
    - the contents of the Pro Git book (i.e. anything with a URL starting with `https://git-scm.com/book` or its PDF versions), which should be raised at [progit/progit2](https://github.com/progit/progit2/issues),
    - Git itself, which should also be raised with the [community](https://git-scm.com/community), or
    - Git for Windows, which should be raised at [git-for-windows/git](https://github.com/git-for-windows/git),
    - and is actually about the website itself (e.g. website-specific content, JS, or CSS that doesn't seem to be doing its job correctly).

**Comment by peff** [comment]
Looks good to me.

**Comment by pedrorijo91** [comment]
Improved as you suggested @sxlijin

**Comment by peff** [comment]
Thanks. Let's get this merged so it can start doing some good. We can make further refinements in another PR if need be.

**Pull request merged**